src/main/java/ui/issuepanel/FilterPanel.java

```diff
@@ -50,11 +50,14 @@
     protected Text nameText;
     protected HBox nameBox;
     protected HBox menuBarNameArea;
+    protected HBox menuBarCloseButton;
     protected Label renameButton;
     protected PanelNameTextField renameTextField;
     protected FilterTextField filterTextField;
     
     private String panelName = "Panel";
+    private static final int TEXT_BOX_WIDTH = PANEL_WIDTH - 15;
+    private static final int NAME_BOX_WIDTH = PANEL_WIDTH - 70;
```

Comment:
These are still very generic names. `NAME_FIELD_WIDTH` and `NAME_DISPLAY_WIDTH`?